packages/components/src/components/popover/popover.tsx

```diff
@@ -73,7 +73,9 @@ const Popover = ({
                             ? is_open ?? ((is_hovered && message) || (is_bubble_hover_enabled && is_bubble_hovered))
                             : is_open ?? (is_hovered && message)) as boolean
                     }
-                    position={alignment}
+                    positions={[alignment]}
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-ignore
                     transitionDuration={0.25}
```

Comment:
need to fix the type here later. for now I don't have solution

Reply:
@iman-fs what kind of error do you get here?

Reply:
I have checked the doc. `transitionDuration`  and `windowBorderPadding` are removed from this package. I was using them and that caused the error. removed them and the error is gone